.github/workflows/publish.yml

```diff
@@ -20,7 +20,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
-      - run: yarn
```

Comment:
This is already included in the `- uses: ./.github/actions/ci-setup` (and that is used one line above this).

Reply:
Does it break publish?

Reply:
I doubt it - it's something that I've noticed when investigating that publish failure.